index.bs

```diff
@@ -1308,52 +1274,47 @@ Instances of <code>WritableStream</code> are created with the internal slots des
 <h4 id="ws-constructor">new WritableStream({ start, write, close, abort, strategy } = {})</h4>
```

Comment:
Yes, good catch, thank you! This actually has normative consequences, as we no longer do destructuring into functions, but instead preserve the whole object.